### PR TITLE
Skip to add an empty step

### PR DESCRIPTION
**Pull request by rgildein**
We do not need to have empty step in plan, so I added the if condition to BaseStep.add step function to skip any empty step.

### CODE REVIEW

**Review by agileshaw — APPROVED** [review]
LGTM

File: tests/unit/steps/test_steps.py [hunk 1]
```diff
@@ -201,6 +203,16 @@ def test_step_add_step():
     """Test BaseStep adding sub steps."""
     exp_sub_steps = 3
     plan = BaseStep(description="plan")
+    for i in range(exp_sub_steps):
+        plan.add_step(BaseStep(description=f"sub-step-{i}", coro=mock_coro()))
+
+    assert len(plan.sub_steps) == exp_sub_steps
+
+
+def test_step_add_step_skiiping_empty():
```
Comment (aieri) [inline tests/unit/steps/test_steps.py:212]:
```suggestion
def test_step_add_step_skipping_empty():
```

**Review by aieri — CHANGES_REQUESTED** [review]
Nice and simple, +1... except for a function name typo, sorry ;)

**Review by gabrielcocenza — CHANGES_REQUESTED** [review]
Small change necessary, otherwise LGTM

**Review by gabrielcocenza — APPROVED** [review]

**Review by agileshaw — APPROVED** [review]

**Review by aieri — APPROVED** [review]

**Pull request merged**